src/views/Landing.js

```diff
@@ -203,19 +242,114 @@ function Landing() {
             </Space>
           </Col>
         </Row>
-        <Row justify="center" style={{ marginTop: '2rem' }} gutter={[24, 16]}>
+        <Row justify="center" style={{ marginTop: '4rem' }} gutter={[24, 16]}>
           {TEMP_FRONTEND_ITEMS.map((feat) => (
             <Col span={4}>
               <FeatureCard feature={feat} />
             </Col>
           ))}
         </Row>
+        <FirstTime />
       </Content>
       <Footer />
     </Layout>
   );
 }
 
+function FirstTime() {
+  let [hasVisited, setVisited] = useState(
+    localStorage.getItem('raiportal:visited')
+  );
+  let setLSHasVisited = () => {
+    localStorage.setItem('raiportal:visited', true);
+  };
```

Comment:
This should also include a call to `setVisited(true)` (need to update both localstorage as well as already loaded state)